src/collection-node-query.js

```diff
@@ -0,0 +1,8 @@
+import imageQuery from './image-query';
+import nodeQuery from './node-query';
+
+const defaultFields = ['id', 'handle', 'updatedAt', 'title', ['image', imageQuery()]];
```

Comment:
Are these fields duplicated in `collectionConnectionQuery`? If so, you should  probably share them.